cli/wallet_test.go

```diff
@@ -45,6 +45,70 @@ func TestWalletAccountRemove(t *testing.T) {
 	require.NoError(t, json.Unmarshal(rawWallet, new(wallet.Wallet)))
 }
 
+func TestWalletChangePassword(t *testing.T) {
+	tmpDir := t.TempDir()
+	e := newExecutor(t, false)
+
+	walletPath := filepath.Join(tmpDir, "wallet.json")
+	e.In.WriteString("acc1\r")
+	e.In.WriteString("pass\r")
+	e.In.WriteString("pass\r")
+	e.Run(t, "neo-go", "wallet", "init", "--wallet", walletPath, "--account")
+
+	e.In.WriteString("acc2\r")
+	e.In.WriteString("pass\r")
+	e.In.WriteString("pass\r")
+	e.Run(t, "neo-go", "wallet", "create", "--wallet", walletPath)
+
+	w, err := wallet.NewWalletFromFile(walletPath)
+	require.NoError(t, err)
+
+	addr1 := w.Accounts[0].Address
+	addr2 := w.Accounts[1].Address
+	t.Run("bad old password", func(t *testing.T) {
+		e.In.WriteString("ssap\r")
```

Comment:
It could fail because new password wasn't provided. We should also add `'pass\rpass\r`.

Reply:
OK, added some.